starfish/image/_registration/_apply_transform/warp.py

```diff
@@ -16,30 +16,29 @@
 
 
 class Warp(ApplyTransformBase):
-    """Class that applies a list of arbitrary skimage GeometricTransforms to an ImageStack
-     using skimage.transform.warp"""
+    """
+    Applies a list of geometric transformations to an ImageStack using skimage.transform.warp
```

Comment:
See @shanaxel42's comments on the other PR on how to link skimage.transform.warp.

Reply:
Addressed/